src/renderers/webgl/WebGLUtils.js

```diff
@@ -147,6 +147,8 @@ function WebGLUtils( gl, extensions, capabilities ) {
 
 		}
 
+		return 0;
```

Comment:
Is this intentional?

Reply:
It was there I think when I diverged to create my branch, see [here](https://github.com/DavidPeicho/three.js/blob/3585caf4f5779c5eb433edd2838257cd65113e33/src/renderers/webgl/WebGLUtils.js#L158).

I guess it's not needed anymore and I miss that when rebasing. Thanks.